docs/src/pages/docs/monoids/First.md

```diff
@@ -138,6 +138,40 @@ concat(concat(a, b), c)
 //=> First( Just "c" )
 ```
 
+#### equals
+
+```haskell
+First a ~> b -> Boolean
+```
+
+Used to compare the underlying values of (2) `First` instances for equality by value, equals takes any given argument and returns `true` if the passed argument is a `First` with an underlying value equal to the underlying value of the `First` the method is being called on. If the passed argument is not a `First` or the underlying values are not equal, equals will return `false`.
```

Comment:
> ... equals takes any given argument and returns `true` if the passed argument is a `First` with an underlying value equal to the underlying value of the `First` the method is being called on.

A little wordy. You can cancel out the second mention of the argument something like

> `equals` is a unary method which returns `true` if its argument is a `First` with an underlying value equal to the underlying value of the `First` the method is being called on. It will return `false` for any other argument.

Reply:
Thanks for the comment, but I guess `equals` is not really unary?